Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN yum -y install epel-release \
     && yum -y install ansible sudo \
     && ansible-galaxy install -p /opt/setup/roles -r requirements.yml
 
-ARG OMERO_VERSION=5.6.1
+ARG OMERO_VERSION=5.6.3
```

Comment:
Bit surprising that this is still on .1 in the mainline.

Reply:
Agreed, the logic for bumping the version seems to happen as part of the tag script - https://github.com/ome/omero-server-docker/blob/master/Makefile#L36 which might explain why we didn't notice it.
But that also means PRs are not tested with the current release so I suspect we want to have this bumped routinely (and maybe the script should perform an assert comparing to the tag rather than a replacement)